packages/Link/index.js

```diff
@@ -17,7 +19,7 @@ export const Link = ({ children, variant = 'primary', ...props }) => {
   )
 
   if (!isChildrenString) {
-    clones = children.map((child, index) => {
+    clones = ensureArray(children).map((child, index) => {
```

Comment:
React le gère de base avec [React.Children.map](https://reactjs.org/docs/react-api.html#reactchildrenmap)
```suggestion
    clones = React.Children.map(children, (child, index) => {
```

Reply:
On dirait que ça ne marche pas :*(

Reply:
trop bien merci @Calvein ❤️